pkgs/top-level/python-packages.nix

```diff
@@ -22311,6 +22338,29 @@ in modules // {
     };
   };
 
+  slob = buildPythonPackage rec {
+    name = "slob-20160304";
```

Comment:
with the new guidelines this will be `name = "slob-unstable-2016-03-04";

https://github.com/NixOS/nixpkgs/blob/36f2dc56493757342e0c86c3df6431bce77eed31/doc/coding-conventions.xml#L260